dockstore-webservice/src/main/java/io/dockstore/webservice/core/Image.java

```diff
@@ -0,0 +1,101 @@
+package io.dockstore.webservice.core;
+
+import java.sql.Timestamp;
+import java.util.HashSet;
+import java.util.Set;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.JoinTable;
+import javax.persistence.OneToMany;
+import javax.persistence.Table;
+
+import io.swagger.annotations.ApiModel;
+import io.swagger.annotations.ApiModelProperty;
+import org.hibernate.annotations.CreationTimestamp;
+import org.hibernate.annotations.UpdateTimestamp;
+
+@Entity
+@ApiModel(value = "Image", description = "Image(s) associated with tags and workflow versions")
+@Table(name = "image")
+@SuppressWarnings("checkstyle:magicnumber")
+
+public class Image {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @ApiModelProperty(value = "Implementation specific ID for the image in this webservice", position = 0)
+    private long id;
+
+    @OneToMany(fetch = FetchType.LAZY)
```

Comment:
I wonder if this shouldn't be eager. My initial thought was what is the use of an image without checksums, but I guess there is other interesting info on the image itself, so I'm not sure.

Reply:
I would go a little further. 
There's a performance hit to an join, but there doesn't seem to enough utility currently to justify a separate table and the cost of the join. If we're always going to be joining checksums and joining all of them, I would prefer an array or a two dimensional array

e.g. `['sha256:abcgdjekgugm'], 'sha256:abcgdjekgugm']`
or `[['sha256', 'abcgdjekgugm'], ['sha256', 'anffygmgum']]`
https://www.postgresql.org/docs/9.1/arrays.html

Reply:
changed to array